### DISCUSSION TITLE
Was heli crashes nerfed?

### DISCUSSION THREAD

**Discussion by Cladicus** [TEHE-Studios/ExpandedHelicopterEvents#129] [Q&A]
Since the feb 20 update I haven't seen a single heli crash, yet before that update there was plenty.
After being frustrated, I tested it on a very rainy game with helis set to insane and after 3 months and maybe 500 helicopters, not a single one crashed.

**Comment by Chuckleberry-Finn** [comment]
This should be an issue report. I'll check into it when I am free, but yes, crash chance was nerfed a while ago.
